src/seedu/addressbook/data/reminder/Reminder.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.reminder;
+
```

Comment:
This class and its non-trivial methods should have header comments. 
If not, developers who write client code for this class will not know how to use it.